Enable azure rbac deployment (#1953)
DFE-Digital/apply-for-qualified-teacher-status: docs/infrastructure.md

$ az account set -s s189-teacher-services-cloud-test
```

> Get access credentials for a managed Kubernetes cluster (passing the afqts env):

Reviewer: ```suggestion
Get access credentials for a managed Kubernetes cluster (in this case for the `development` environment):
```

We don't really use the acronym `afqts` so this could be confusing for developers on the team.

Author: updated as suggested
